.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+name: Release
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out
+        uses: actions/checkout@v4
+
+      - name: Publish to Hex.pm
+        uses: erlangpack/github-action@v3
+        env:
+          HEX_API_KEY: ${{ secrets.HEX_API_KEY }}
```

Comment:
This secret is not set. Can you please setup it? Is there a way to add it once in Infisical, and then reference it in more than one place? (to avoid cloning the same secret N times and having to spend half day to update all instances when it rotates).

Reply:
This is possible, given our current setup. @san983 feel free to reach out if you need assistance.

Reply:
Sure thing @DXTimer